modules/openapi-generator/src/main/resources/python/python-experimental/api.mustache

```diff
@@ -45,48 +45,80 @@ class {{classname}}(object):
         self.api_client = api_client
 {{#operation}}
 
-        def __{{operationId}}(self, {{#sortParamsByRequiredFlag}}{{#allParams}}{{#required}}{{paramName}}, {{/required}}{{/allParams}}{{/sortParamsByRequiredFlag}}**kwargs):  # noqa: E501
+        def __{{operationId}}(
+            self,
+{{#requiredParams}}
+{{^defaultValue}}
+            {{paramName}},
+{{/defaultValue}}
+{{/requiredParams}}
+{{#requiredParams}}
+{{#defaultValue}}
+            {{paramName}}={{{defaultValue}}},
+{{/defaultValue}}
+{{/requiredParams}}
+            **kwargs
+        ):
```

Comment:
This multiline style comes from the python black style tool and is valid per pep8
Spreading params out to one per line lets us delete the `# noqa: E501` here